app/src/test/java/by/academy/jc/DirtyTest.java

```diff
@@ -0,0 +1,25 @@
+package by.academy.jc;
+
+import by.academy.jc.shorokhov.Dirtycalculator;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DirtyTest {
+    Dirtycalculator calculator = new Dirtycalculator();
+    double sum = calculator.sum(1.5,4);
+    double testSum = 5.5;
+    double sub = calculator.sub(2.5,3);
```

Comment:
Don't place business logic in the class initialization section, except for cases like `@PostConstruct` or `@Before / @BeforeEach`.